test-conf/tests/drivers/flash/common/tt_blackhole_tt_blackhole_smc.conf

```diff
@@ -2,3 +2,4 @@ CONFIG_FLASH=y
 CONFIG_MSPI=y
 CONFIG_SPI=n
 CONFIG_FLASH_MSPI_NOR_RUNTIME_PROBE=y
+CONFIG_HEAP_MEM_POOL_SIZE=8192
```

Comment:
You could probably just add this to the board configuration file. It's easier than adding it in so many other places.